src/WebMotions.Fake.Authentication.JwtBearer/HttpClientExtensions.cs

```diff
@@ -78,6 +82,27 @@ public static HttpClient SetFakeBearerToken(this HttpClient client, string usern
 
             return client;
         }
+        
+        /// <summary>
+        /// Set a fake bearer token in form of a JWT. 
+        /// </summary>
+        /// <param name="client"></param>
+        /// <param name="claims"></param>
+        /// <returns></returns>
+        public static HttpClient SetFakeJwtBearerToken(this HttpClient client, Dictionary<string, object> claims)
+        {
+            var tokenHandler = new JwtSecurityTokenHandler();
+            var tokenDescriptor = new SecurityTokenDescriptor
+            {
+                Subject = new ClaimsIdentity(),
+                Expires = DateTime.UtcNow.AddDays(7),
+                Claims = claims
+            };
+
+            var token = tokenHandler.CreateToken(tokenDescriptor);
+            var jwt = tokenHandler.WriteToken(token);
+            return client.SetToken("FakeBearer", jwt);
```

Comment:
As an enhancement, I just realized now, could you replace all the `"FakeBearer"` in the file by `FakeJwtBearerDefaults.AuthenticationScheme`? Would be great!